src/relay/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+- Change to use the new [JSX transforms](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html)
```

Comment:
Are there any implications for the user-land code? Does it require to use the new JSX transform as well?

Reply:
No, it does not. But it needs to be using a version compatible with the new `JSX` transform. So 16.14.0 or greater

Reply:
The transpilde code looks like 

```jsx
import {jsx as _jsx} from 'react/jsx-runtime';

function App() {
  return _jsx('h1', { children: 'Hello world' });
}
```

So as long as that import path exists in react, the consumer is fine.